command/commands/user.py

```diff
@@ -97,23 +96,63 @@ def handle(self, command, user_id, channel):
             self.delete_helper(user_id, args.slack_id, channel)
 
         elif args.which == "edit":
-            # stub
-            msg = "user edited: "
-            if args.member is not None:
-                msg += "member: {}, ".format(args.member)
-            if args.name is not None:
-                msg += "name: {}, ".format(args.name)
-            if args.email is not None:
-                msg += "email: {}, ".format(args.email)
-            if args.pos is not None:
-                msg += "position: {}, ".format(args.pos)
-            if args.github is not None:
-                msg += "github: {}, ".format(args.github)
-            if args.major is not None:
-                msg += "major: {}, ".format(args.major)
-            if args.bio is not None:
-                msg += "bio: {}".format(args.bio)
-            return msg
+            param_list = {
+                "member": args.member,
+                "name": args.name,
+                "email": args.email,
+                "pos": args.pos,
+                "github": args.github,
+                "major": args.major,
+                "bio": args.bio,
+            }
+            self.edit_helper(user_id, param_list, channel)
+
+    def edit_helper(self, user_id, param_list, channel):
+        """
+        Edit user from database.
+
+        If param_list[0] is not None, edits user with
+        ID param_list[0], else edits user with user_id
+
+        :param user_id: Slack ID of user who is calling the command
+        :param param_list: List of user parameters that are to be edited
+        :param channel: ID of Slack channel that called command
+        :return: returns error message if not admin and command
+                   edits another user, returns edit message if user is edited
+        """
+        edited_user = None
+        if param_list["member"] is not None:
+            try:
+                admin_user = self.facade.retrieve_user(user_id)
+                if admin_user.get_permissions_level() != Permissions.admin:
+                    return self.bot.send_to_channel(self.permission_error,
+                                                    channel)
```

Comment:
Don't know if this is intentional, but `bot.send_to_channel` does not return anything; you will be returning a `None` value instead.

Reply:
I put that there so that the method would stop running there, idk if there's a better way of doing that though